pkg/baremetal/v1alpha3/register.go

```diff
@@ -0,0 +1,46 @@
+package v1alpha3
```

Comment:
A comment for later, but I'd maybe keep the `v1alpha1` packages here for a grace period for being able to convert objects over eventually.

A comment for now: this is an api, it should exist under `pkg/apis`